Src/VimCore/CoreInterfaces.fs

```diff
@@ -2612,6 +2612,9 @@ type ModeArgument =
     /// Enter command mode with a partially entered command and then return to normal mode
     | PartialCommand of Command: string
 
+    /// Cancel any operation that is in-progress, such as refactoring
+    | CancelOperation

```

Comment:
Generally `ModeArgument` is an option to the new mode in a switch where as `CancelOperation` is somewhat an argument to the previous mode. Think we should change the name / comment to make this clear. Maybe `CancelPreviousModeOperations`.

Reply:
I don't think so. It is wrong to say that the operation being cancelled belongs to or is any way related to the previous mode. The previous mode is just some random mode that happened to be the current mode when the user pressed `<C-c>`. Conceptually the "cancel operation" mode argument is informing the VimBuffer of the cancellation, not the old mode or the new mode.